hugolib/site.go

```diff
@@ -380,6 +380,7 @@ type SiteInfo struct {
 	BuildDrafts           bool
 	canonifyURLs          bool
 	relativeURLs          bool
+	IsServer              bool
```

Comment:
I think what we want to do is just add a method that returns the value of this{

https://github.com/gohugoio/hugo/blob/3cdf19e9b7e46c57a9bb43ff02199177feb55768/deps/deps.go#L206

Reply:
Or more specifc this value:

Reply:
Ah gotcha. Will rework.